data/json/items/tool/musical_instruments.json

```
@@ -61,6 +61,28 @@
       ]
     }
   },
+  {
+    "id": "wood_flute",
+    "type": "TOOL",
+    "name": { "str": "wooden flute" },
+    "copy-from": "bone_flute",
+    "material": [ "wood" ],
+    "color": "brown",
+    "use_action": {
+      "type": "musical_instrument",
+      "volume": 12,
+      "fun": -5,
+      "fun_bonus": 2,
+      "speed_penalty": 10,
+      "description_frequency": 20,
+      "player_descriptions": [
+        "You play a little tune on your flute.",
+        "You play a beautiful piece on your flute.",
+        "You play a piece on your flute that resembles the howling wind through Aurignacian caves.",
+        "Your bone flute wails for the fallen during the hunt."
```

```suggestion
        "Your flute wails for the fallen during the hunt."
```
Unfortunate that the lines are all the same as the bone one but oh well.